MyGet.ps1

```diff
@@ -0,0 +1,43 @@
+# Performs the build for MyGet and uploads the generated NuGet packages to
+# the "google-dotnet" feed.
+#
+# MyGet's build syncs the repo and subrepos, to build manually be sure to:
+# git submodule update --init ClientGenerator
+
+function PrintHeader($message) {
+    Write-Host "****************************************"
+    Write-Host $message
+    Write-Host "****************************************"
+}
+
+Write-Host ".NET Client Libraries, build shard $Env:BuildShard"
+
+# Just needs to be a version installed on the system.
+[Environment]::SetEnvironmentVariable(
```

Comment:
$Env:VisualStudioVersion = "14.0" should work. We just need this to be set locally.